src/Translation/TranslationDelegate.php

```diff
@@ -34,7 +34,7 @@ public function translate(string $text, ?string $source_language, string $target
     return null;
   }
 
-  private function validateLanguage(?string $language): void
+  protected function validateLanguage(?string $language): void
```

Comment:
Are you testing invalid language codes? If not, this may not be needed

Reply:
I am not directly testing the language codes in the behat test, but I think the mocked translate function should still behave similarly to the real translate function and validate the language codes since the tests do execute an API call that will include the source and target language. So, with the protected function, if the language codes are invalid in the API call within the test then the test will fail. I can remove the checks if you think they are excessive though, I just thought that it wouldn't be bad to have an extra built-in check.

Reply:
I think the mock class should only mock behaviour that is needed by the test. We can add validation in the future if we need to test that. Test/mock classes are always overhead for maintenance, so less is better

Reply:
Okay, I will remove that check then